services/docsrs/docsrs.service.js

```diff
@@ -54,7 +54,7 @@ export default class DocsRs extends BaseJsonService {
   async handle({ crate, version = 'latest' }) {
     const { build_status: buildStatus } = (
       await this.fetch({ crate, version })
-    ).pop()
+    ).shift()
```

Comment:
Would you mind changing this to just destructure the first item in the array? I realize we were already mutating it and that in practice this will typically only have one element, but there's really no need to mutate the collection the first place